src/index.html

```diff
@@ -11,7 +11,146 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form 
+      action = "https://mate-academy-form-lesson.herokuapp.com/create-application" method = "POST"
+    >
+      <fieldset class = "group">
+        <legend>Personal information</legend>
+        <label class="formField">Surname:
+          <input
+            type = "text"
+            name = "surname"
+            required
+            autocomplete="off"
+            class = "inpMb"
+          >
+        </label>
+        <label class="formField">Name:
+          <input
+            type = "text"
```

Comment:
```suggestion
            type="text"
```

redundant spaces, fix everywhere